spark/src/test/scala/org/apache/spark/sql/benchmark/CometAggregateBenchmark.scala

```diff
@@ -64,13 +64,7 @@ object CometAggregateBenchmark extends CometBenchmarkBase {
           spark.sql(query).noop()
         }
 
-        benchmark.addCase(s"SQL Parquet - Comet (Scan) ($aggregateFunction)") { _ =>
-          withSQLConf(CometConf.COMET_ENABLED.key -> "true") {
-            spark.sql(query).noop()
-          }
-        }
```

Comment:
It no longer makes sense to benchmark Comet with `Scan` vs `Scan & Exec` configurations, especially for aggregates. Comet scans won't accelerate aggregates.